common-ui/src/main/res/values/design-system-typograhy.xml

```diff
@@ -34,7 +35,7 @@
     <style name="TextAppearance.DuckDuckGo.Headline2">
         <item name="android:textSize">20sp</item>
         <item name="android:lineSpacingExtra">6sp</item>
-        <item name="android:fontFamily">sans-serif-medium</item>
+        <item name="android:fontFamily">sans-serif</item>
```

Comment:
I edited this value in the design review of bottom sheets because the title (h2) wasn't looking as it is expected. On Figma, h2, h3, h4 and h5 are medium 🤔

Reply:
I'll check in with Sveta, she reviewed this again... 🤷

Reply:
Wait, this is the legacy typography, `Typography.DuckDuckGo.H2` has  `<item name="android:fontFamily">sans-serif-medium</item>`